### ISSUE TITLE
Focus indicator does not display correctly when keyboard focus is obtained after a single legend text line feed

### ISSUE THREAD

**Issue by YoungLdocT** [highcharts/highcharts#19335]
#### Expected behaviour

1. When a single legend receives keyboard focus, the focus indicator should be clearly visible.
2. Legend text on the card should not be truncated when reflowed at 1280*1024 400% resolution.

#### Actual behaviour

1.Focus indicator does not display correctly when keyboard focus is obtained after a single legend text line feed.Here focus indicator lands on 2 legends on overlaps with legends text at 400% zoom.
![image](https://github.com/highcharts/highcharts/assets/68530410/61e3b8c8-15db-4c6a-bd19-c7cee0cbd566)

2.Legends text truncated at 'Top senders and recipients' card at reflow in 400% zoom.
![image](https://github.com/highcharts/highcharts/assets/68530410/882755ff-8a27-40a1-baf0-baa913258e18)

#### Live demo with steps to reproduce

https://codepen.io/lydoc1/pen/rNQYEvL

#### Product version

highcharts 9.0.0
#### Affected browser(s)
Checked in Edge

**Comment by msftedad** [comment]
Hi, Any update on this issue? Please provide the ETA to fix this issue.

**Comment by msft-rahul** [comment]
Hi @TorsteinHonsi Thanks for fixing where can we verify this fix, can you share fix URL? as it does not fix our side.

**Comment by msft-rahul** [comment]
Hi @TorsteinHonsi Thanks for fixing can you share Highchart example URL where we can check this fix as it is not fixed our side
